include/cantera/core.h

```diff
@@ -0,0 +1,23 @@
+/**
+ * @file core.h
+ *
+ * Support for Cantera core calculations from C++ application programs.
+ * This header file includes several headers needed to create and use objects
+ * which evaluate thermo properties, chemical kinetics and transport properties.
+ */
+
+#ifndef CT_INCL_CORE_H
+#define CT_INCL_CORE_H
+
+#include "cantera/base/Solution.h"
+#include "cantera/thermo/ThermoPhase.h"
+#include "cantera/kinetics/Kinetics.h"
+
+#ifndef CT_SKIP_TRANSPORT
+// TransportFactory facilitates instantiation of alternative transport objects
+// @todo implement convenient way to switch transport model within Solution objects
+//      so dependency can be simplified to transport/TransportBase.h
+#include "cantera/transport/TransportFactory.h"
+#endif
```

Comment:
I don't think we should do this. This precludes transport calculations in the same file as a Reactor Net if zerodim.h is included. Does this significantly improve compilation time?

Reply:
I don't think that it has any major effects. Will take it out ...